docs/docs/concepts/intervals-and-windows.md

```diff
@@ -123,10 +123,14 @@ dependencies: []
 
 Notice the window configuration is specified under field `task.window`. **Important** note, the `version` field decides which
 version of window capability to be used. Currently available window version are `version: 1`, `version: 2` and `version: 3`. Version 3 is recommended
-to be used as version 1 will soon be deprecated. To know the difference between the two version, run the `playground` feature for window:
+to be used as version 1 will soon be deprecated. To play around with window configuration, try `playground` feature for window:
```

Comment:
`version 1 and 2 will soon be deprecated`

Reply:
updated as suggested, with a little modification